composer.json

```diff
@@ -22,7 +23,8 @@
         "openeuropa/code-review": "~1.6",
         "openeuropa/drupal-core-require-dev": "^8.9 || ^9.1",
         "openeuropa/oe_media": "^1.14",
-        "openeuropa/oe_paragraphs": "^1.10",
+        "openeuropa/oe_multilingual": "^1.8",
+        "openeuropa/oe_paragraphs": "dev-master",
```

Comment:
After the meeting with Francesco about testing in oe_showcase I think oe_paragraph, oe_media, etc., must be moved to require section but please confirm with Francesco.